### ISSUE TITLE
Questions about Midas Training

### ISSUE THREAD

**Issue by Only-bottle** [antocad/FocusOnDepth#3]
Hello,

First of all, I would like to say thank you for your work.
I'm going to retraining the MiDas model. However, [MiDas repository](https://github.com/isl-org/MiDaS) does not have a train code.
Is it possible to train the MiDas model using your train code? 
Or have you ever trained the MiDas model?
The dataset is NYU v2.

Thank you

**Comment by younesbelkada** [comment]
Hi,

Thank you for your message, 
To the best of our knowledge, the current implementation is for DPT, I am not sure if DPT and MiDas are the same architecture. I will double check that,
If this is not the case, I think that it is possible to train MiDas, but you have to come up with your own implementation of it, and replace the variable model at [Trainer.py](https://github.com/antocad/FocusOnDepth/blob/e693cd39947195374714375e5f466f46ee7b8028/FOD/Trainer.py#L24). If you can come up with your implementation feel free to send us a Pull Request and we will review it
Thanks!

**Comment by Only-bottle** [comment]
Thank you for your kind reply.

I will proceed by referring to the "Trainer.py" you mentioned.

I have a question.

Your [Loss Function](https://github.com/antocad/FocusOnDepth/blob/master/FOD/Loss.py) has ScaleAndShiftInvariantLoss. 
I would like to know if you are referring to the link below. 
[Loss function of MiDas](https://gist.github.com/dvdhfnr/732c26b61a0e63a0abc8a5d769dbebd0)

Thanks!

**Comment by younesbelkada** [comment]
Hi,

Thank you very much for your message. I can confirm you that we use the same loss as MiDas as we have referred to the link that you have shared.
That would be great ! Looking forward to your implementation

**Comment by jiaqixuac** [comment]
Hi,

Thanks for the great repo.
For the loss, I wonder if the target is in the disparity space.
And if it is applicable to use the median version for training.
https://gist.github.com/ranftlr/1d6194db2e1dffa0a50c9b0a9549cbd2

Thanks!
